### DISCUSSION TITLE
Broken links?

### DISCUSSION THREAD

**Discussion by bobturneruk** [pykale/pykale#167] [Q&A] [answered]
The colab and binder buttons don't work when clicked here:

https://github.com/pykale/pykale/blob/main/examples/digits_dann_lightn/tutorial.ipynb

**Comment by bobturneruk** [comment]
...and this seems to be the only route to the notebooks through the docs. (Windows 10, Chrome 91.0.4472.114)

**Reply by haipinglu** [reply]
Thank you @bobturneruk !
The links are all right. I can open them if right-click and open in _new window_. 

I guess the problem is that it tries to open that link **within the notebook**. Can you force the link to open in new window when clicked?
Googled:
https://stackoverflow.com/questions/34446172/open-url-in-new-tab-from-ipython-notebook-jupyter-cell?rq=1
https://stackoverflow.com/questions/39415653/jupyter-notebook-open-link-in-new-tab

**Reply by bobturneruk** [reply]
Will be resolved by https://github.com/pykale/pykale/issues/172

**Comment by bobturneruk** [comment, marked as answer]
Any examples of this working in other repositories? I'm having a hard time fixing this! Seems like images can't be clickable links in GitHub's rendering...

**Reply by haipinglu** [reply]
I can not (easily) find. If it is not solvable in affordable time. My suggestion is to make it just a normal hyperlink in the notebook, without the nice icons (images), just a link like this [Open in Colab](https://colab.research.google.com/github/pykale/pykale/blob/main/examples/digits_dann_lightn/tutorial.ipynb). This should work fine.

**Reply by haipinglu** [reply]
Even slightly better, it could save one line because you could put them side by side rather than one on top of the other.

**Reply by bobturneruk** [reply]
I was literally doing this as you were typing!
